src/main.rs

```diff
@@ -24,7 +28,7 @@ fn main() {
 
     let opengl = OpenGL::V3_2;
 
-    let mut window: PistonWindow =
+    let mut window: GlutinWindow =
```

Comment:
Does using `GlutinWindow` limit us in anyway? I thought I rememberedd reading something about how `PistonWindow` would use whichever backend was appropriate for the target automatically.